setup.py

```diff
@@ -13,6 +13,7 @@
   'dataframe': ['numpy >= 1.13.0', 'pandas >= 0.21.0', 'toolz >= 0.7.3',
                 'partd >= 0.3.10', 'cloudpickle >= 0.2.1'],
   'distributed': ['distributed >= 1.22'],
+  'diagnostics': ['bokeh >= 0.12.3'],
```

Comment:
In https://github.com/conda-forge/dask-feedstock/blob/master/recipe/meta.yaml, we require bokeh >=0.13.0

Reply:
Thanks Tom. I think I copied this from some reference in the codebase here. Will bump the version to match the feedstock.

Reply:
Updated. Please recheck.